backend/account_v2/tests/test_last_admin_guard.py

```
@@ -0,0 +1,228 @@
+
+    @classmethod
+    def get_user_by_email(cls, email: str) -> SimpleNamespace | None:
+        return cls._member_by_email
+
+    @classmethod
+    def get_members(cls) -> list:
+        return cls._all_members
+
+    # Kept for compatibility with other call sites in the controller.
+    @classmethod
+    def get_user_by_id(cls, id: object) -> None:
```

Ruff (0.15.15)

[error] Function argument `id` is shadowing a Python builtin

(A002)